mantidimaging/gui/windows/operations/presenter.py

```diff
@@ -106,6 +111,9 @@ def filter_uses_parameter(self, parameter):
             self.model.params_needed_from_stack is not None else False
 
     def do_apply_filter(self):
+        if self.view.safeApply.isChecked():
+            self.original_images_stack = self.stack.presenter.images.copy()
+
         self.view.clear_previews()
         apply_to = [self.stack]
         if self.stack.presenter.images.has_proj180deg():
```

Comment:
Having a 180 degree projection loaded causes an error as it tries to open the compare dialog for it as well